Sources/WebDriver/Session.swift

```diff
@@ -30,16 +31,34 @@ public class Session {
             owned: true)
     }
 
-    public static func createW3C(webDriver: any WebDriver, alwaysMatch: Capabilities, firstMatch: [Capabilities] = []) throws -> Session {
+    /// Initializer for W3C Protocol
+    private convenience init(webDriver: any WebDriver, alwaysMatch: Capabilities, firstMatch: [Capabilities]) throws {
         let response = try webDriver.send(Requests.Session_W3C(
             alwaysMatch: alwaysMatch, firstMatch: firstMatch))
-        return Session(
+        self.init(
             webDriver: webDriver,
             existingId: response.value.sessionId,
             capabilities: response.value.capabilities,
             owned: true)
     }
 
+    public convenience init(webDriver: any WebDriver, capabilities: Capabilities) throws {
```

Comment:
How about this signature being:

```swift
init(driver:capabilities:)
```

Do you need to have any capabilities? Does it make sense to have a Session without any extensions and therefore have no special capabilities. If so, it might be worth considering:

```swift
init(driver: any WebDriver, capabilities: Capabilities? = nil)
```